### PR TITLE
Debug code

### PR DESCRIPTION
**Pull request by jmtsuji**
Fixed two minor code bugs in the `develop` branch (typos) and updated docs.

### CODE REVIEW

File: backblast [hunk 1]
```diff
@@ -395,7 +395,7 @@ function perform_auto() {
   local pident
   pident=25
   local qcov
-  pident=50
```
Comment (jmtsuji) [inline backblast:398]: Bug 1 - wrong variable name

File: snakemake/Snakefile [hunk 2]
```diff
@@ -126,7 +126,7 @@ if config.get("phylogenetic_tree_newick") == "subjects":
         output:
             "phylogeny/iqtree_out.treefile"
         conda:
-            "envs/gtotree.yaml"
```
Comment (jmtsuji) [inline snakemake/Snakefile:129]: Bug 2 - `envs` was moved back a folder relative to the Snakefile

File: snakemake/Snakefile [hunk 3]
```diff
@@ -139,7 +139,7 @@ if config.get("phylogenetic_tree_newick") == "subjects":
         shell:
             "GToTree -A {input} -H {params.phylogenetic_model} -o phylogeny/gtotree -T IQ-TREE "
                 "-c {params.sequence_length_threshold} -G {params.minimum_hit_fraction} "
-                "-n {threads} -j {threads} > {log} 2>&1 && "
```
Comment (jmtsuji) [inline snakemake/Snakefile:142]: Minor enhancement to print log statements from this particular program to the screen so the user can monitor progress (can take time to run).

File: README.md [hunk 4]
```diff
@@ -2,46 +2,44 @@ BackBLAST_Reciprocal_BLAST
...
+Non-orthologous genes are filtered out by identifying and extracting only bidirectional best BLASTP hits using a graph-based algorithm. The algorithm is illustrated below:
 
 ![BackBLAST Algorithm](https://raw.githubusercontent.com/LeeBergstrand/BackBLAST-Gene-Cluster-Finder/master/Media/BackBLAST-Algorithm.gif)
 
-- **Visualization** - This repository also includes tools for visualizing the results from **BackBLAST_core.py** in the form of a R heatmap. Here is an example:
-
-![Example Results](https://raw.githubusercontent.com/LeeBergstrand/BackBLAST-Gene-Cluster-Finder/master/Media/ExampleResults.jpeg)
+Furthermore, `backblast` allows users to visualize the results from bidirectional BLASTP in the form of a heatmap. Here is an example:
 
+![Example Results](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41396-020-0650-2/MediaObjects/41396_2020_650_Fig7_HTML.png)
```
Comment (jmtsuji) [inline README.md:17]: Added figure generated by BackBLAST2 rather than BackBLAST1

File: README.md [hunk 5]
```diff
@@ -2,46 +2,44 @@ BackBLAST_Reciprocal_BLAST
 ==========================
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3465955.svg)](https://doi.org/10.5281/zenodo.3465955)
 
-Copyright Lee H. Bergstrand and Jackson M. Tsuji, 2019
```
Comment (jmtsuji) [inline README.md:5]: Generally updated the README with details for BackBLAST2

File: README.md (hunk 4 above)
Comment (LeeBergstrand) [reply README.md:17]: Good

**Pull request merged**